tests/VolleyManagement.Specs/PlayersContext/CreatePlayerSteps.cs

```diff
@@ -85,11 +115,121 @@ public void ThenNewPlayerShouldBeSuccesfullyAdded()
             PlayerEntity actualPlayer;
             using (var ctx = TestDbAdapter.Context)
             {
-                actualPlayer = ctx.Players.SingleOrDefault(p => p.Id == _addedPlayerId);
+                actualPlayer = ctx.Players.SingleOrDefault(p => p.Id == _player.Id);
             }
 
             actualPlayer.Should().NotBe(null, "Player should be saved to the DB");
             actualPlayer.Should().BeEquivalentTo(_player);
         }
+
+        [Then(@"InvalidEntityException is thrown")]
+        public void ThenInvalidEntityExceptionIsThrown()
+        {
+            _exception.Should().BeOfType(typeof(InvalidEntityException));
+        }
+
+        [When(@"I execute QuickCreatePlayer")]
+        public void WhenIExecuteQuickCreatePlayer()
+        {
+            try
+            {
+                foreach (var player in _playersForQuickCreate)
```

Comment:
This scenario should cover another method in service

![image](https://user-images.githubusercontent.com/13673187/39798502-7c1e6360-5369-11e8-9de7-0c3747596bbc.png)

Reply:
Ok